### PR TITLE
SQL safety

### PR DESCRIPTION
**Pull request by markstory**
Add additional information on unsafe query usage.

Refs cakephp/cakephp#7157

### CODE REVIEW

File: en/appendices/3-1-migration-guide.rst [hunk 1]
```diff
@@ -40,6 +40,24 @@ RoutesShell
...
+  * theme
+  * autoLayout
+  * viewPath
+  * layoutPath
+
+  Instead of setting these properties on your controllers, you should set them
+  on the view using methods with matching names::
+
+    // In a controller
+    $this->getView()->layout('advanced');
```
Comment (antograssiot) [inline en/appendices/3-1-migration-guide.rst]: don't you think it's worth showing an example of the old syntax, just to highlight the `"you should set them on the view using methods with matching names"`. I was not confused but I can already see people not getting the idea...

Appart from this, looks good

Comment (HavokInspiration) [inline en/appendices/3-1-migration-guide.rst]: I agree with @antograssiot. A before / after could be a good thing.

File: fr/appendices/3-1-migration-guide.rst [hunk 2]
```diff
@@ -44,6 +44,24 @@ Controller
...
+  * viewPath
+  * layoutPath
+
+  Au lieu de définir ces propriétés dans vos controllers, vous devrez les
+  définir dans votre View en utlisant des méthodes de même nom::
+
+    // Dans un controller
+    $this->getView()->layout('advanced');
+
+Ces méthodes doivent être appelée après que vous ayez déterminé quelle classe
```
Comment (HavokInspiration) [inline fr/appendices/3-1-migration-guide.rst]: appelées*

File: fr/orm/query-builder.rst [hunk 3]
```diff
@@ -1166,6 +1166,35 @@ requête en utilisant ``query()``::
 Généralement, il est plus facile de supprimer les données en utilisant les
 entities et :php:meth:`~Cake\\ORM\\Table::delete()`.
 
+Prévention contre les Injections SQL
+====================================
+
+Alors que l'ORM et les couches d'abstraction de base de données empêchent la
+plupart des problèmes relatifs aux injections SQL, il est toujours possible que
+vous soyez vulnérables face à une utilisation abusive. Lorsque vous utilisez
```
Comment (HavokInspiration) [inline fr/orm/query-builder.rst]: I'd say "utilisation incorrecte" or "mauvaise utilisation".

**Comment by HavokInspiration** [comment]
Wouldn't it be good to provide counter examples of how to safely do the not-safe queries ?

**Pull request merged**